### PR TITLE
JS Error while creating new applicaiton  fix

### PR DESCRIPTION
**Pull request by kdsinghcoder**
To fixing : #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +64,6 @@ The user sees the following screen when during login
   - Inside [Handlebar](https://handlebarsjs.com/)
 - Database 
   - [MongoDB](https://docs.mongodb.com/) via [Mongoose](https://mongoosejs.com/)
+  
+
```
Comment (YashKumarVerma) [inline README.md:68]: empty lines not needed

File: src/user/user.schema.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Application } from 'src/application/application.schema';
 import { Document } from 'mongoose';
 
-export type UserDocument = User & Document;
+export type UserDocument = User & Document;
```
Comment (YashKumarVerma) [inline src/user/user.schema.ts:10]: i think we can undo this change.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
Required changes are done.

File: src/user/user.schema.ts [hunk 3]
```diff
@@ -7,8 +7,6 @@ import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Application } from 'src/application/application.schema';
 import { Document } from 'mongoose';
 
-export type UserDocument = User & Document;
-
```
Comment (YashKumarVerma) [inline src/user/user.schema.ts:11]: this line should not be removed.

earlier there was an extra space, only that had to be removed.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
add again.

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
Fixes #17

**Pull request merged**